cmd/ore/aliyun/create-image.go

```diff
@@ -63,6 +63,20 @@ func init() {
 }
 
 func runCreate(cmd *cobra.Command, args []string) error {
+	// Check if image exists first when force not enabled
+	if !force {
+		images, err := API.GetImages(name)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "getting images: %v", err)
+			os.Exit(1)
```

Comment:
Unrelated, though why do `Fprintf` then `os.Exit` manually here if we're using `RunE`? Can't we just return an error and have cobra print that?

Reply:
Yeah just returning would work, was mostly keeping to the old style that had been done in the past for other commands.